Automattic/newspack-plugin
# Disable the MC4WC campaign tracking cookie

### All Submissions:


### Changes proposed in this Pull Request:

A performance concern is cookies that are set for logged out users. These cause variations in requests that lead to too many uncached responses. It seems that the Mailchimp for WooCommerce plugin sets a cookie for every visitor regardless of whether it's actually needed. This change stops that behaviour and helps to reduce the possibility of uncached page views.

Note that I'm introducing a new concept with this too - that of compatibility fixes for known plugins - so thoughts welcome on that as an approach if there's a better way.

### How to test the changes in this Pull Request:

1. Load any page of a Newspack site using the MC4WC plugin and observe that a `mailchimp_landing_site` cookie is set
2. Apply this PR, then clear all site data
3. Load the page again and observe that the cookie is no longer set

### Other information:
